src/stream/FFmpegStream.cpp

```diff
@@ -729,14 +732,6 @@ bool FFmpegStream::Open(bool fileinfo)
     m_streaminfo = true;
   }
 
-  if (iformat && (strcmp(iformat->name, "mov,mp4,m4a,3gp,3g2,mj2") == 0))
-  {
-    CURL url(m_streamUrl);
-    //if (URIUtils::IsRemote(strFile))
-    if (!url.GetProtocol().empty() && !url.IsProtocol("file"))
-      m_pFormatContext->iformat->flags |= AVFMT_NOGENSEARCH;
-  }
-
```

Comment:
Why remove this?

Reply:
@AlwinEsch did this in core PR https://github.com/xbmc/xbmc/pull/21248/commits/aec31f55115df5d62905688d0cd2dbe3ccfab1ac#diff-bad369057589a59119615d8f380bff839b51c2100f2e8b5fd92b24fb61ea3fc6R546

Reply:
@AlwinEsch do you recall the reason for this?